Client.xcodeproj/project.pbxproj

```diff
@@ -5212,7 +5214,6 @@
 				2816F0001B33E05400522243 /* UIConstants.swift in Sources */,
 				E650755F1E37F756006961AC /* Try.m in Sources */,
 				3B6889C51D66950E002AC85E /* UIImageColors.swift in Sources */,
-				EB7A652020699C6100B52A5F /* WebPagesForTesting.swift in Sources */,
```

Comment:
Has this been removed for some reason? There are some tests using the this server here...

Reply:
Hm, I don't recall changing anything here